accounts/abi/bind/auth.go

```diff
@@ -91,12 +91,12 @@ func MakeTransactOpts(accountKey *ecdsa.PrivateKey, nonce *big.Int, gasLimit uin
 }
 
 // MakeTransactOptsWithKeystore creates a transaction signer with nonce, gasLimit, and gasPrice from a keystore wallet.
-func MakeTransactOptsWithKeystore(wallet accounts.Wallet, from common.Address, nonce *big.Int, chainID *big.Int, gasLimit uint64, gasPrice *big.Int) *TransactOpts {
-	if wallet == nil {
+func MakeTransactOptsWithKeystore(ks *keystore.KeyStore, from common.Address, nonce *big.Int, chainID *big.Int, gasLimit uint64, gasPrice *big.Int) *TransactOpts {
```

Comment:
How about merging `NewKeyedTransactorWithKeystore` and `MakeTransactOptsWithKeystore`?

Reply:
Like `NewKeyedTransactor` and `MakeTransactOpts`, should be need each functions?